magefiles/util.go

```diff
@@ -21,8 +21,8 @@ func goTest(path string, args ...string) error {
 
 // run go test in a directory
 func goDirTest(dir string, path string, args ...string) error {
-	testArgs := append([]string{"test", "-failfast", "-count=1"}, args...)
-	return RunSh(goCmdForTests(), WithV(), WithDir(dir), WithArgs(testArgs...))(path)
+	testArgs := append([]string{"test", path, "-failfast", "-count=1"}, args...)
+	return RunSh(goCmdForTests(), WithV(), WithDir(dir))(testArgs...)
```

Comment:
The tests were unable to access the `-enablePgbouncer` flag if it came before the path for some reason. After looking at this, it didn't feel like there was any reason that the path needed to be treated as a unique argument. Now I treat them similarly and place path at the front.